src/lib/types/permissions.ts

```diff
@@ -33,6 +33,8 @@ export const READ_API_TOKEN = 'READ_API_TOKEN';
 export const UPDATE_TAG_TYPE = 'UPDATE_TAG_TYPE';
 export const DELETE_TAG_TYPE = 'DELETE_TAG_TYPE';
 export const UPDATE_FEATURE_VARIANTS = 'UPDATE_FEATURE_VARIANTS';
+export const UPDATE_FEATURE_ENVIRONMENT_VARIANT =
+    'UPDATE_FEATURE_ENVIRONMENT_VARIANTS';
```

Comment:
Seems like a typo where we have plural on the value and singular on the const itself - Make them consistent?

Reply:
Good eyes! Thank you, fixed